cfme/tests/physical_infrastructure/test_physical_server_details.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.physical.provider.lenovo import LenovoProvider
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="module")]
+
+
+@pytest.fixture(scope="module")
+def physical_server(appliance, provider, setup_provider_modscope):
+    # Get and return the first physical server
+    physical_servers = appliance.collections.physical_servers.all(provider)
+    yield physical_servers[0]
+
+
+def test_physical_server_details(physical_server):
+    """Navigate to the physical server details page and verify that the page is displayed"""
+    physical_server_view = navigate_to(physical_server, 'Details')
+    assert physical_server_view.is_displayed
```

Comment:
**Optional**

The navigate_to() method (CFMENavigateStep.go()) takes a `wait_for_view` boolean kwarg that toggles a `wait_for` of `view.is_displyed`. You could use this to roll the assert up, though that has a bit of a downside for test readability.

Mostly FYI for some of the navigation capabilities :)